proto/tendermint/abci/types.proto

```diff
@@ -132,22 +144,24 @@ message RequestPrepareProposal {
 
 message Response {
   oneof value {
-    ResponseException          exception            = 1;
-    ResponseEcho               echo                 = 2;
-    ResponseFlush              flush                = 3;
-    ResponseInfo               info                 = 4;
-    ResponseInitChain          init_chain           = 5;
-    ResponseQuery              query                = 6;
-    ResponseBeginBlock         begin_block          = 7;
-    ResponseCheckTx            check_tx             = 8;
-    ResponseDeliverTx          deliver_tx           = 9;
-    ResponseEndBlock           end_block            = 10;
-    ResponseCommit             commit               = 11;
-    ResponseListSnapshots      list_snapshots       = 12;
-    ResponseOfferSnapshot      offer_snapshot       = 13;
-    ResponseLoadSnapshotChunk  load_snapshot_chunk  = 14;
-    ResponseApplySnapshotChunk apply_snapshot_chunk = 15;
-    ResponsePrepareProposal    prepare_proposal     = 16;
+    ResponseException           exception            = 1;
+    ResponseEcho                echo                 = 2;
+    ResponseFlush               flush                = 3;
+    ResponseInfo                info                 = 4;
+    ResponseInitChain           init_chain           = 5;
+    ResponseQuery               query                = 6;
+    ResponseBeginBlock          begin_block          = 7;
+    ResponseCheckTx             check_tx             = 8;
+    ResponseDeliverTx           deliver_tx           = 9;
+    ResponseEndBlock            end_block            = 10;
+    ResponseCommit              commit               = 11;
+    ResponseListSnapshots       list_snapshots       = 12;
+    ResponseOfferSnapshot       offer_snapshot       = 13;
+    ResponseLoadSnapshotChunk   load_snapshot_chunk  = 14;
+    ResponseApplySnapshotChunk  apply_snapshot_chunk = 15;
+    ResponsePrepareProposal     prepare_proposal     = 16;
+    ResponseExtendVote          extend_vote          = 17;
+    ResponseVerifyVoteExtension verify_vote_extension = 18;
```

Comment:
Does the formatter have to be re-ran to make the equals signs align, or is this fine per golint?

Reply:
the developer needs to install some extra tooling. We can leave as is and fix the little things like this later on